src/lib/tabs/tab-group.spec.ts

```diff
@@ -70,6 +70,25 @@ describe('MdTabGroup', () => {
       });
     }));
 
+    it('should set to correct tab on fast change', async(() => {
+      let component = fixture.componentInstance;
+      component.selectedIndex = 0;
+      fixture.detectChanges();
+
+      setTimeout(() => {
```

Comment:
Why is this `setTimeout` instead of using `fixture.whenStable()`?

Reply:
I used `setTimeout` because thats what his demo used.